### PR TITLE
Setup Blog App

### PR DESCRIPTION
**Pull request by omar-muhamad**
In this PR I did:

- [ ] [Install rails new app and make it work](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/dad1f89824906b45878bc2bbc438f7109b858235).
- [ ] [Add all URLs with routes, controllers, and views](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/ed5bf458d6297169220fe98d89ae50d196fe45f4).
- [ ] [Add linters and fix linters errors](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/68f79a8b1568071070b0df7b7c92ad6407a87cb1).
- [ ] [Edit README file](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/59bf3daca917872c3cca014e88ce18d8e1e372e8).

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: :show, only: [:index, :show]
```
Comment (andersonlebon) [inline config/routes.rb]: ## [OPTIONAL]

- [ ] Nice job on putting routes however I can see a repeated permission for the show route for a post `resources :posts, only: :show, only: [:index, :show]` kindly correct it with `resources :posts, only: [:index, :show]`

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+- Install Rails by running this order in your terminal `gem install rails` (good luck with that).
+
+## Setup
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Type `rails s` to run live server.
+5. Open your browser on this " [127.0.0.1:3000](127.0.0.1:3000) ".
+
```
Comment (andersonlebon) [inline README.md]: - [ ] Kindly provide all instructions on how to run this project locally(`installing bundle, setting up the database`)

**Review by andersonlebon — COMMENTED** [review]
## Hi @Omar-Muhamad 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-muhamad** [comment]
Github actions not working right now but all linters passed locally with me.

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+## Getting started
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Install all gems by run this command `bundle install`
+5. Open `database.yml` file inside `./config` folder and put your username and password.
+6. Type `rails s` to run live server.
+7. Open your browser on this [127.0.0.1:3000](127.0.0.1:3000)
+8. Now you will see a button saying `create database` click on it and turn off the rails sever an run it again.
```
Comment (julie-ify) [inline README.md:28]: - [ ] **[OPTIONAL]** Good job so far, however, kindly add instructions like run `rails db:create` and `rails db:migrate` to create and migrate the database then run `rails db:seed` to seed data if there's any. This is more explained instructions on how to set up a database.

**Review by julie-ify — APPROVED** [review]
Hi @Omar-Muhamad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**